Source/WebCore/css/process-css-properties.py

```diff
@@ -3361,7 +3361,7 @@ def _generate_animation_property_value_setter(self, to, property):
         to.write(f"if (is<CSSValueList>(value)) {{")
         to.write(f"    // Walk each value and put it into an animation, creating new animations as needed.")
         to.write(f"    for (auto& currentValue : downcast<CSSValueList>(value)) {{")
-        to.write(f"        if (childIndex <= list.size())")
+        to.write(f"        if (childIndex >= list.size())")
```

Comment:
Can we make this consistent with https://searchfox.org/wubkat/rev/114aa2c8594807cbc34dd5dff48a9e2addfd1e00/Source/WebCore/css/process-css-properties.py#3350 by swapping the order there?

Reply:
I'll just fix the bug, there are all sorts of cleanups that could be done here.